skeletor/build/js.config.js

```diff
@@ -7,9 +7,9 @@ module.exports = {
 			name: '@deg-skeletor/plugin-rollup',
```

Comment:
[fix] Sorry I missed this the first time I reviewed, but you need to _add_ a rollup for config for legacy bundles rather than _replace_ the rollup config for modern bundles. As you have it now, this configuration will only build legacy bundles.